build/build.go

```diff
@@ -575,7 +575,7 @@ func NewMappingCallback(m *sourcemap.Map, goroot, gopath string) func(generatedL
 	}
 }
 
-func jsFilesFromDir(dir string) ([]string, error) {
+func JsFilesFromDir(dir string) ([]string, error) {
```

Comment:
> Some more documentation would also be nice, however in this case I would like to avoid giving the impression that the `build` package is stable API. It does not feel ready to me. If we add more docs then we should warn in the package description about potential future changes.

Good point, agreed.